org.scala-ide.sdt.debug/src/org/scalaide/debug/internal/breakpoints/ScalaDebugBreakpointManager.scala

```diff
@@ -88,6 +96,8 @@ private[debug] object ScalaDebugBreakpointManagerActor {
   case class BreakpointAdded(breakpoint: IBreakpoint)
   case class BreakpointRemoved(breakpoint: IBreakpoint)
   case class BreakpointChanged(breakpoint: IBreakpoint, delta: IMarkerDelta)
+  case class ReenableBreakpointsAfterHcr(classNames: Seq[String])
+  case object ReenableBreakpointAfterHcr
```

Comment:
Is this case object used? Even if it is used, can't the same semantics be attached to the case class, when `classNames` is empty?